lib/configDefaults.js

```diff
@@ -11,3 +15,16 @@ module.exports = {
             'https://github.com/dhis2/dhis2-demo-db/blob/master/sierra-leone/{version}/dhis2-db-sierra-leone.sql.gz?raw=true',
     },
 }
+
+let userConfig
+try {
+    userConfig = require(configFile)
+    reporter.debug(`User configuration loaded from: ${configFile}`)
+} catch (e) {
+    reporter.debug(`Failed to load user config from: ${configFile}`)
```

Comment:
Might be nice not to call it "failing" if the file DNE?  But maybe that's too much overhead to check existence first?

Reply:
Hm, the `fs` module in Node recommends this strategy:

>Using `fs.access()` to check for the accessibility of a file before calling `fs.open()`, `fs.readFile()` or `fs.writeFile()` is not recommended. Doing so introduces a race condition, since other processes may change the file's state between the two calls. Instead, user code should open/read/write the file directly and handle the error raised if the file is not accessible.